experimenter/experimenter/features/management/commands/load_feature_configs.py

```diff
@@ -22,7 +26,31 @@ def handle(self, *args, **options):
             NimbusFeatureConfig.objects.values_list("application", "slug")
         )
 
-        for feature in Features.all():
+        # A mapping of (application, slug) to feature configs.
+        #
+        # Features will include multiples of the same feature, each with a
+        # different version. We only need to create a single NimbusFeatureConfig
+        # for each of those, so if we keep track of them we can skip a bunch of
+        # update_or_create() calls.
+        feature_configs: dict[tuple[str, str], NimbusFeatureConfig] = {}
+
+        # Iterate through the unversioned features first for the initial
+        # update_or_create() calls. The unversioned features are the most
+        # up-to-date (since they come from the main branch (or equivalent)), so
+        # lets use the descriptions from there.
+        #
+        # Then we can iterate over the versioned feature and create any that
+        # aren't in the unversioned manifests.
+        #
+        # When we are ingesting versioned Features, we want to update the
+        # NimbusFeatureConfig objects with the most up-to-date description.
+        for feature in itertools.chain(
+            Features.unversioned(),
+            sorted(Features.versioned(), key=lambda f: f.version, reverse=True),
+        ):
+            if (feature.application_slug, feature.slug) in feature_configs:
+                continue
```

Comment:
So is the idea here that we go over the unversioned ones first, and if we hit every feature then that's sufficient, but there may be some feature for which there is no unversioned schema but there may be a versioned schema, and so in that case we fall back to that?

Reply:
Right. Say Firefox 121 removes feature A. Then when we are on 121 nightly, the unversioned experimenter.yaml won't have feature A, but the versioned experimenter.yaml for 120, 120.0.1, etc. will have it listed.